### PR TITLE
Add license info of jquery-one-page-nav, cc #5500

### PR DESCRIPTION
**Pull request by sufuf3**
Reference: https://github.com/davist11/jQuery-One-Page-Nav/blob/master/jquery.nav.js#L6

### CODE REVIEW

**Comment by sufuf3** [comment]
@kennynaoh please help me review it, thanks!

File: ajax/libs/jquery-one-page-nav/package.json [hunk 1]
```diff
@@ -24,5 +24,9 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/davist11/jQuery-One-Page-Nav"
-  }
+  },
+  "licenses": [
+    "MIT",
+    "GPL"
```
Comment (kennynaoh) [inline ajax/libs/jquery-one-page-nav/package.json]: @sufuf3 Please make sure which version of `GPL`, thanks!

**Comment by PeterDaveHello** [comment]
Any updates here?

**Comment by sufuf3** [comment]
@kennynaoh I confirm the lib of the license, please help me review this PR again, thanks!
Ref: https://github.com/davist11/jQuery-One-Page-Nav/issues/159

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@sufuf3 @kennynaoh @pvnr0082t do not forget to review the title (and the first comment) please, thanks.

**Pull request merged**